lib/notes.js

```diff
@@ -1,22 +1,30 @@
-const add = action => {
-    const note = {
-        id: 1,
-        text: action.payload
-    };
-    return note;
-};
 
-const execute = action => {
-    if (action.type === 'add' || action.type === 'a') {
-        return add(action);
-    } else {
-        return {
-            error: 'crap balls'
+class Note {
+
```

Comment:
constructor(text, id) {
    this.id = id || Date.now();
    this.text = text;
  }